src/verify.v3.integration.test.ts

```diff
@@ -131,7 +150,7 @@ describe("verify v3(integration)", () => {
         ]
       `);
       expect(valid).toBe(true);
-    });
+    }, 10000);
```

Comment:
are these tests slower now after the PR?

Reply:
yeah, some test will timeout so need to extend the timeout for the tests.